website/models.py

```
@@ -1965,16 +1965,38 @@ class GitHubReview(models.Model):
     body = models.TextField(null=True, blank=True)
     state = models.CharField(max_length=50)  # e.g., "APPROVED", "CHANGES_REQUESTED", "COMMENTED"
     submitted_at = models.DateTimeField()
     url = models.URLField()
 
+    class Meta:
+        constraints = [
+            models.CheckConstraint(
+                check=models.Q(reviewer__isnull=False) | models.Q(reviewer_contributor__isnull=False),
+                name="at_least_one_reviewer",
+            ),
+        ]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)